src/main/java/SpringJPA/Model/UserType.java

```diff
@@ -1,8 +1,8 @@
 package SpringJPA.Model;
 
 public enum UserType {
-    TRIAL,
-    NONTRIAL,
-    PREMIUM,
-    ADMIN
+    ROLE_TRIAL,
+    ROLE_NONTRIAL,
+    ROLE_PREMIUM,
+    ROLE_ADMIN
 }
```

Comment:
Why did the role names need to be changed?

Reply:
The role names needed to be changed so that they could be recognized as roles by Spring Security. The naming convention used always has 'ROLE_' before the role name, so this needed to be done to implement the security for the admin pages.